.pre-commit-config.yaml

```diff
@@ -25,7 +25,7 @@ repos:
     hooks:
       - id: code-format
         name: code-format
-        entry: bash -c './gradlew spotlessCheck || (echo ""; echo "!!Automatically running spotlessApply!!"; ./gradlew spotlessApply)'
+        entry: bash -c './gradlew spotlessCheck || (echo ""; echo "!!Automatically running spotlessApply!!"; ./gradlew spotlessApply && git add -u)'
```

Comment:
Could `git add -u` inadvertently stage changes to files that are not intended to be added to the commit?

Reply:
It could. I can add a git status with it so the user is aware of this, thoughts?

Reply:
Decided to take the git add -u and add the git status instead.